### PR TITLE
MAUI Calendar Getting Started Content Modified

### PR DESCRIPTION
**Pull request by MohanrajSF3559**
**Description:**
MAUI Calendar Getting Started Content Modified

### CODE REVIEW

File: MAUI/Calendar/getting-started.md [hunk 1]
```diff
@@ -7,23 +7,45 @@ control: SfCalendar
 documentation: ug
 ---
 
-# Getting started with .NET MAUI Calendar (SfCalendar)
+# Getting Started with the .NET MAUI Calendar
+
 This section explains how to add the [.NET MAUI Calendar](https://www.syncfusion.com/maui-controls/maui-calendar) control. This section covers only the basic features needed to get started with Syncfusion Calendar.
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/getting-started.md:13]: Follow the steps below to add a basic scheduler view to your project.

- Add this line

File: MAUI/Calendar/getting-started.md [hunk 2]
```diff
@@ -7,23 +7,45 @@ control: SfCalendar
...
+1. In **Solution Explorer,** right-click the project and choose **Manage NuGet Packages.**
+2. Search for [Syncfusion.Maui.Calendar](https://www.nuget.org/packages/Syncfusion.Maui.Calendar/) and install the latest version.
+3. Ensure the necessary dependencies are installed correctly, and the project is restored.
 
-4. Initialize `SfCalendar.`
+## Step 3: Register the handler
+
+The [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
 
 {% tabs %}
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/getting-started.md:50]: Check code snippets forRegister the handler - seems xaml file also added

File: MAUI/Backdrop/getting-started.md [hunk 3]
```diff
@@ -56,8 +74,9 @@ namespace MyProject
 }
 
 {% endhighlight %}
+{% endtabs %}
 
-## Initializing backdrop page
+## Step 4: Add a Basic Backdrop view.
```
Comment (MuniappanSubramanian) [inline MAUI/Backdrop/getting-started.md:79]: Backdrop view. ---> Backdrop page

File: MAUI/Calendar/getting-started.md (hunk 1 above)
Comment (MohanrajSF3559) [reply MAUI/Calendar/getting-started.md:13]: Added

File: MAUI/Calendar/getting-started.md (hunk 2 above)
Comment (MohanrajSF3559) [reply MAUI/Calendar/getting-started.md:50]: Now changed

File: MAUI/Backdrop/getting-started.md (hunk 3 above)
Comment (MohanrajSF3559) [reply MAUI/Backdrop/getting-started.md]: addressed

File: MAUI/Backdrop/getting-started.md [hunk 4]
```diff
@@ -9,42 +9,59 @@ documentation: ug
 
 # Getting started with .NET MAUI Backdrop Page
 
-This section explains the steps required to add the Backdrop page and its elements such as BackLayer and FrontLayer in the project. This section covers only the basic features needed to know and gets started with the Syncfusion backdrop page.
+This section explains the steps required to add the Backdrop page and its elements such as BackLayer and FrontLayer in the project. This section covers only the basic features needed to know and gets started with the Syncfusion backdrop page. Follow the steps below to add a basic scheduler view to your project.
```
Comment (MuniappanSubramanian) [inline MAUI/Backdrop/getting-started.md:13]: Update control name as back drop page

File: MAUI/Barcode-Generator/getting-started.md [hunk 5]
```diff
@@ -9,40 +9,59 @@ documentation: ug
 
 # Getting Started with .NET MAUI Barcode Generator
 
-This section explains the steps required to add the [`.NET MAUI Barcode Generator`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.html) control. This section explains the steps required to add the barcode and set its symbology. This section covers only basic features needed to get started with Syncfusion barcode generator control.
+This section explains the steps required to add the [`.NET MAUI Barcode Generator`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.html) control. This section explains the steps required to add the barcode and set its symbology. This section covers only basic features needed to get started with Syncfusion barcode generator control. Follow the steps below to add a basic calendar view to your project.
```
Comment (MuniappanSubramanian) [inline MAUI/Barcode-Generator/getting-started.md:12]: control name check -

File: MAUI/Calendar/getting-started.md [hunk 6]
```diff
@@ -7,36 +7,93 @@ control: SfCalendar
...
-{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3 5" %}
+{% highlight XAML hl_lines="3 5" %}
 
 <ContentPage   
     . . .
-    xmlns:calendar="clr-namespace:Syncfusion.Maui.Calendar;assembly=Syncfusion.Maui.Calendar">
+    xmlns:scheduler="clr-namespace:Syncfusion.Maui.Calendar;assembly=Syncfusion.Maui.Calendar">
 
-    <calendar:SfCalendar />
+    <scheduler:SfCalendar />
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/getting-started.md:92]: in calendar mentioned as **scheduler**  - update correctly

File: MAUI/CircularProgressBar/getting-started.md [hunk 7]
```diff
@@ -9,42 +9,59 @@ documentation: ug
 
 # Getting started with .NET MAUI Circular ProgressBar
 
-This section explains the steps required to add the circular progress bar control with the progress and its customizable elements such as indeterminate, segment, progress fill, and track fill. This section covers only the basic features needed to know and gets started with the Syncfusion circular progress bar.
+This section explains the steps required to add the circular progress bar control with the progress and its customizable elements such as indeterminate, segment, progress fill, and track fill. This section covers only the basic features needed to know and gets started with the Syncfusion circular progress bar. Follow the steps below to add a basic calendar view to your project.
```
Comment (MuniappanSubramanian) [inline MAUI/CircularProgressBar/getting-started.md:12]: changes control name

File: MAUI/DateTimePicker/getting-started.md [hunk 8]
```diff
@@ -7,51 +7,39 @@ control: SfDateTimePicker
 documentation: ug
 ---
 
-# Getting started with .NET MAUI Date Time Picker (SfDateTimePicker)
-This section explains how to add the Date Time Picker control. It covers only the basic features needed to get started with Syncfusion Date Time Picker.
+# Getting Started with the .NET MAUI DateTimePicker
+This section explains how to add the Date Time Picker control. It covers only the basic features needed to get started with Syncfusion Date Time Picker. Follow the steps below to add a basic calendar view to your project.
```
Comment (MuniappanSubramanian) [inline MAUI/DateTimePicker/getting-started.md:11]: chgnage control name

File: MAUI/Barcode-Generator/getting-started.md (hunk 5 above)
Comment (MohanrajSF3559) [reply MAUI/Barcode-Generator/getting-started.md]: Changed

File: MAUI/Calendar/getting-started.md (hunk 6 above)
Comment (MohanrajSF3559) [reply MAUI/Calendar/getting-started.md]: Changed

File: MAUI/CircularProgressBar/getting-started.md (hunk 7 above)
Comment (MohanrajSF3559) [reply MAUI/CircularProgressBar/getting-started.md]: Changed

File: MAUI/DateTimePicker/getting-started.md (hunk 8 above)
Comment (MohanrajSF3559) [reply MAUI/DateTimePicker/getting-started.md]: Changed

File: MAUI/Backdrop/getting-started.md (hunk 4 above)
Comment (MohanrajSF3559) [reply MAUI/Backdrop/getting-started.md:13]: Updated

**Review by MuniappanSubramanian — APPROVED** [review]

**Pull request merged**